src/platform/Linux/ConnectivityManagerImpl.cpp

```diff
@@ -146,6 +164,16 @@ CHIP_ERROR ConnectivityManagerImpl::_Init()
         ChipLogError(DeviceLayer, "Failed to reset WiFi statistic counts");
     }
 #endif
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+    pmWiFiPAF = &WiFiPAF::WiFiPAFLayer::GetWiFiPAFLayer();
+    pmWiFiPAF->Init(&DeviceLayer::SystemLayer());
+
+    struct sigaction sa = {};
+    sa.sa_handler       = StopSignalHandler;
+    sa.sa_flags         = SA_RESETHAND;
+    sigaction(SIGINT, &sa, nullptr);
```

Comment:
Please, do not install any signal handlers in the SDK code. Signal handler (if needed) should be installed by the application which uses SDK, not the other way.

https://github.com/project-chip/connectedhomeip/issues/38944